doc/audit/builtin_maps.rst

```diff
@@ -143,6 +143,15 @@ Versions of the code allowed to join the current network.
 
 .. note:: On versions ``3.0.0-dev1`` and below, value was an enum with one possible value ``ALLOWED_TO_JOIN``
 
+``nodes.security_policies``
+~~~~~~~~~~~~~~~~~~
+
+Security policies that nodes can run and join the current network.
+
+**Key** Security Policy Digest: Hash of the security policy string which is reported by the SNP attestation.
+
+**Value** Raw Security Digest: The raw string of the security policy
```

Comment:
We need a link to something that explains what a security policy is.

Reply:
I spoke to Ken and he said there isn't any public documentation yet, I can keep an eye on it and add it when there are some?

Reply:
@DomAyre sounds good! I think a note above this saying this is experimental would be good.